### ISSUE TITLE
Show the result in a web graph and keep the history result

### ISSUE THREAD

**Issue by zhihuiFan** [brianfrankcooper/YCSB#781]
I think the ideal load test tool should have the following features.

1.  Support many different kind of databases,  MongoDB, Cassandra, Couchbase at least.
2.  User don’t need to any code,  but user can configure different JDBC options,  like maxPoolSize,  socketTimeOut and so on.
3.  It should have a WEB Ui,  then people don’t need to login some server to run command line.
4.  The final result should be showed in a graph way.   
5.  Both test case and final result should be keep automatically and can be review in future easily
6.  We can write some comments for a test case,  it should be also kept for future review. 
 
YCSB  can be used for purpose 1, 2 perfectly,  but 3,4,5,6 are not supported. 

Do we have such roadmap?    thanks

**Comment by busbey** [comment]
What do you mean by "in a graph way"? If you mean charting the results, currently folks use post-processing tools to chart what they're interested in. In particular, if you use the feature to export HDRHistogram log files, then you can use [that projects web-based plotting tool](http://hdrhistogram.github.io/HdrHistogram/plotFiles.html) to make charts.

A service that can launch jobs and store results would be a nice addition. Is that something you'd be interested in working on?

**Comment by zhihuiFan** [comment]
Yes, I mean charting the result.  I will learn how to do that currently. 

I absolutely like to contribute to YSCB,  that's a really cool stuff!   But due to my current workload,  my schedule has be used up in the next 5 months.   If everything goes well,  I will spend time on this if this tasks is not assign to others.  and I will make the load test for new user case as a service.

**Comment by hzruandd** [comment]
It‘s cool！it is wonderful to show the result in a web graph or draw up the graphs by those testing-results in mongoDB.
